src/main/java/alvioneurope/gsergeev/javagroup/clientlogimporter/MethodInfo.java

```diff
@@ -8,8 +8,84 @@
     private String methodName;
     private String sourcePath;
     private Integer lineNumber;
+    private Boolean methodInfo;
 
     public MethodInfo(String source) {
-
-    };
+        this.methodInfo=splitString(source);
+    }
+    public boolean checkingClosedBrackets(String string){
+        if((string.indexOf('(')!=-1&&string.indexOf(')')!=-1)||(string.indexOf('(')==-1&&string.indexOf(')')==-1))
+            return true;
+        else
+            if(string.indexOf('(')!=-1&&string.indexOf(')')==-1)
+                return false;
+        return false;
+    }
+    public boolean checkingStringToAnInt(String string){
```

Comment:
??? It isn't good idea check Integer by loop.
By the way. Is the 7234623546325645324532542 correct int?